### DISCUSSION TITLE
when optimize the yolov8 in Optuna, some trial not respond at some epoch

### DISCUSSION THREAD

**Discussion by BinZhou-23** [optuna/optuna#4517] [Q&A]
When I optimize in optuna, why the training process does not respond at some epoch and does not end, can I use try except to throw this error and move on to the next trial. if so, how can I write it?

![image](https://user-images.githubusercontent.com/77983100/225284000-cf69c8bb-754e-42b3-a053-af76701af2bd.png)

and my code is:
![image](https://user-images.githubusercontent.com/77983100/225286096-e00c34d5-fcc8-4fe6-937b-d7e4fa35f8be.png)
![image](https://user-images.githubusercontent.com/77983100/225286279-33155b22-b058-4466-b4a9-b57071e54202.png)

**Comment by nzw0301** [comment]
did you see https://github.com/ultralytics/ultralytics/issues/673? I think Optuna code would not be a source of the error.
